api_specifications/case-api-0.1.0.yaml

```
@@ -1499,3 +1554,178 @@ components:
+
+    JoinCaseV2Request:
+      type: object
+      properties:
+        RequestInfo:
+          $ref: >-
+            https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/RequestInfo
+        joinCaseData:
+          $ref: '#/components/schemas/JoinCaseDataV2'
+
+    JoinCaseDataV2:
+        type: object
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 8

(indentation)